esp/src/eclwatch/templates/WUDetailsWidget.html

```diff
@@ -36,6 +36,7 @@
                                         <option value="High">${i18n.High}</option>
                                     </select>
                                     <input id="${id}AllowForeignFiles" title="${i18n.AllowForeignFiles}:" name="AllowForeignFiles" colspan="2" data-dojo-props="trim: true" checked data-dojo-type="dijit.form.CheckBox" />
+                                    <input id="${id}UpdateSuperFiles" title="${i18n.UpdateSuperFiles}:" name="UpdateSuperFiles" colspan="2" data-dojo-props="trim: true" data-dojo-type="dijit.form.CheckBox" />
```

Comment:
I'll remove the trim.  I thought it might be better to default to unchecked since this is an overwrite behavior.  Can always change it later if it turns out people always turn it on anyway.